### DISCUSSION TITLE
How to put an image in the header like on these 2 sites

### DISCUSSION THREAD

**Discussion by HDInfautre** [templaza/astroid-framework#366] [Q&A]
hello
Can we create a similar header with the Astroid theme from https://ascension.fr/ or https://www.altiplano-voyage.com/ ? 
Regards

**Comment by Chacapamac** [comment]
https://efoilsurf.ca/en/

**Reply by HDInfautre** [reply]
Hello
Ok thanks, is there a tutorial for this?
Or how it was done step by step?

**Reply by Chacapamac** [reply]
![Screen Shot 2023-08-03 at 1 18 24 PM](https://github.com/templaza/astroid-framework/assets/1907276/426cd1c3-dc81-4f21-ab80-3a5693ed4867)

**CSS**
**Section TOPBAR**
```
body.frontpage section#topbar {
  z-index: 8;
  position: absolute !important;
  width: 100% !important;
  height: 40px !important;

```

**Section HEADER**
```
body.frontpage section#header-section {
  position: absolute !important;
  margin-top: 40px !important;

```
**Section VIDEO Slideshow**
```
#hero-section {
  margin-top: -1px !important;
```

Maybe a little more tweaking to make it work on all devices, but in general, this is it.

**Reply by HDInfautre** [reply]
Hello

Thank you for this configuration.
I'm having several problems
In Header Section, I've added a row with 2 columns: [Logo module] + [main menu].
![Astroid-layouts](https://github.com/templaza/astroid-framework/assets/6817936/08edbe4c-2166-47ea-9f8e-f4902e10dff9)
I've added the css in the custom code section

**1/ Logo module**
I created a "logo_acc" position in joomla
I've refreshed the caches, but I can't select it in astroid [Logo module].
![joomla-module-logo](https://github.com/templaza/astroid-framework/assets/6817936/0d57fdf8-489a-4e51-81d6-7498ba6c55ab)

******
![Astroid-element-logo](https://github.com/templaza/astroid-framework/assets/6817936/2ccae698-4b74-44c3-88e9-91f335191108)

**2/ main menu**
in [main menu], I've selected the "menu" position but it doesn't appear properly.
What's more, I still have the old menu linked to astroid.
![Astroid-element-menu](https://github.com/templaza/astroid-framework/assets/6817936/59b5dac6-1934-4c63-a3d9-3c7c66f65d1e)

**3/ position: absolute**
I couldn't find the section or element where I could declare this?

**4/ photo**
I don't understand which photo should be reworked?
![Astroid-colonne-image](https://github.com/templaza/astroid-framework/assets/6817936/1a3efb04-b7e5-441f-aa72-be771f6ccaa3)
https://j4test.88h.ovh/

Regards

**Comment by Chacapamac** [comment]
@HDInfautre 
**We start with the Logo Module**

> **1/ Logo module**
> I created a "logo_acc" position in joomla
> I've refreshed the caches, but I can't select it in astroid [Logo module].

_By the way, I’m French, born in Cannes, and my father immigrated to Canada. In Quebec now!_

• Ok, You created the position by saving a custom module with the custom position “logo_acc”?
- After, You should be able to select it in a Astroid custom Position????? Weird.
- - Did you try **"logo-acc"** Not sure if the underline is accepted????

Also, and maybe even better, you can use the normal **Astroid Logo options** in the **Astroid Template Manager**
Also here, you CSS all that to your liking and it is easier to manage the logo directly in the **Astroid Template Manager**

![Screen Shot 2023-08-04 at 3 57 21 PM](https://github.com/templaza/astroid-framework/assets/1907276/44152489-f84f-4dcc-8e19-28012eb57638)

On my exemple, I use a custom module, but in my newer websites I use Astroid Logo Options.

**Comment by HDInfautre** [comment]
Bonjour @Chacapamac 
Ah le Québec, coup de coeur de beaucop de français y compris moi :-) 
Super puis-je vous parler en français ? car mon anglais est mauvais surtout en regardant via un traducteur

**1/ logo**
En fait je n'avai pas bien vidé tous les caches. J'ai vérifier les syntaxes Logo_acc ou Logo-acc fonctionne :-)
Maintenant j'ai mis le logo vias astroid

Pouvez-vous me répondre à mes autres problèmes 2, 3 et 4/ ?
Bonne journée
Hervé

**Comment by Chacapamac** [comment]
Bonjour @HDInfautre 

> **2/ main menu**
> in [main menu], I've selected the "menu" position but it doesn't appear properly.
> What's more, I still have the old menu linked to astroid.

**Ici je ne suis pas sur de comprendre ce que tu veut faire?**
Par défaut, le menu (et mobile menu) qui est choisi pour apparaître dans le “Header” et le “Main Menu” de Joomla?

![Screen Shot 2023-08-07 at 7 24 02 PM](https://github.com/templaza/astroid-framework/assets/1907276/32651ebf-65df-4d3d-a555-6b8db3dc1e6e)

![Screen Shot 2023-08-07 at 7 31 32 PM](https://github.com/templaza/astroid-framework/assets/1907276/537b7017-c195-4d7d-95b1-a7a52b745ddb)

**Comment by HDInfautre** [comment]
Bonjour
Merci pour ce retour
**2/ menu principal**
Effectivement j'ai supprimé ce quze j'avai cru comprendre au début (I've added a row with 2 columns: [Logo module] + [main menu].)  et je n'ai plus ce double menu :-)

**3/ position: absolute**
I couldn't find the section or element where I could declare this?
=> je n'ai toujotus pas vu où? et si je dois faire cela ?

4/ photo header
enfin le plus important est que l'image n'est pas du tout comme je le souhaiterai, comme:
https://ascension.fr/ OU https://www.altiplano-voyage.com/
L'image épouse tout le haut du site comme on peut voir l'image : https://www.altiplano-voyage.com/bundles/altiplanovoyage/img/img-header-home.jpg

Voici ma configuration dans la ligne du header-section > Hero image :
![Capture d’écran du 2023-08-08 11-34-40](https://github.com/templaza/astroid-framework/assets/6817936/a9696db0-9077-4db3-9d4e-45dd66b27867)
![Capture d’écran du 2023-08-08 11-31-59](https://github.com/templaza/astroid-framework/assets/6817936/ae40c7d4-79fd-4a29-a0fa-cba08be84676)

**Comment by HDInfautre** [comment]
Hello
I don't know if @Chacapamac is still available now. ?
Certainly part of it is in French, but if anyone could help me finish this, I'm still stopped.
Regads

**Comment by Chacapamac** [comment]
Sorry, @HDInfautre , really busy these days.
• If I understand well, you fix the problem **2/ Main Menu**

• **3/ position: absolute**
I couldn't find the section or element where I could declare this?

— Personally I always use a “custom SCSS file (You can also use a “Custom CSS” file) or you can enter your generic CSS directly in each layouts you want to modify.
![Screen Shot 2023-08-10 at 10 55 53 AM](https://github.com/templaza/astroid-framework/assets/1907276/801ffad1-d5a4-4775-b2cf-8a68e6af2140)

— In a **Custom CSS**
Just add a Plain Text File named “template.css” inside the folder **Joomla root ➔ media ➔ templates ➔ site ➔ yourtemplatefoldername ➔ css folder**

— In a **Custom SCSS**
_**SCSS is my preferred choice, you can use plain CSS here but the SCSS offer more powerful way to write the code** (especially the variables)_ — https://www.w3schools.com/sass/sass_variables.php
Just add a Plain Text File named “custom.css” inside the folder **Joomla root ➔ media ➔ templates ➔ site ➔ yourtemplatefoldername ➔ scss folder**

**• 4/ photo header**
I suggest you fix the CSS first and give us a screenshot of the image problem.
